File: SupplierPortal.Web/Modules/İtem/Items/ItemsColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace SupplierPortal.İtem.Columns;

[ColumnsScript("İtem.Items")]
[BasedOnRow(typeof(ItemsRow), CheckNames = true)]
public class ItemsColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    public short ItemCode { get; set; }
    [EditLink]
    public string ItemName { get; set; }
}

File: SupplierPortal.Web/Modules/ServerTypes/İtem/ItemsColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { ItemsRow } from "./ItemsRow";

export interface ItemsColumns {
    Id: Column<ItemsRow>;
    ItemCode: Column<ItemsRow>;
    ItemName: Column<ItemsRow>;
}

export class ItemsColumns extends ColumnsBase<ItemsRow> {
    static readonly columnsKey = 'İtem.Items';
    static readonly Fields = fieldsProxy<ItemsColumns>();
}